tests/src/JIT/CodeGenBringUpTests/LocallocB_N.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+
+using System;
+using System.Runtime.CompilerServices;
+public class BringUpTest
+{
+    const int Pass = 100;
+    const int Fail = -1;
+
+    // Reduce all values to byte
+    public static unsafe bool CHECK(byte check, byte expected) {
+        return check == expected;
+    }
+
+    [MethodImplAttribute(MethodImplOptions.NoInlining)]
+    public static unsafe int LocallocB_N(int n)
+    {
+        byte* a = stackalloc byte[n];
+
+        for (int i = 0; i < n; i++)
+        {
+            a[i] = (byte) i;
+        }
+
+        for (int i = 0; i < n; i++)
+        {
+            if (!CHECK(a[i], (byte) i)) return i;
+        }
+
+        return -1;
+    }
+
+    public static int Main()
+    {
+        int ret;
+
+        ret = LocallocB_N(1);
+        if (ret != -1) {
+            Console.WriteLine("Failed on index: " + ret);
+            return Fail;
```

Comment:
This comment applies to all your test files.

---
In reply to: [109272951](https://github.com/dotnet/coreclr/pull/10609#discussion_r109272951) [](ancestors = 109272951)